packages/web-components/src/components/inline-loading/inline-loading.ts

```diff
@@ -28,24 +28,66 @@ export { INLINE_LOADING_STATE };
 @customElement(`${prefix}-inline-loading`)
```

Comment:
```suggestion
/**
 * Lnline loading spinner.
 *
 * @element cds-inline-loading
 * @fires cds-inline-loading-onsuccess The custom event fired when inline-loading has finished status
 */
@customElement(`${prefix}-inline-loading`)
```

Adding this so it gets captured in the attributes, properties, events table in storybook docs